source/chapters/appendix.rst

```diff
@@ -311,10 +297,26 @@ Option                  Description
                         * Doesn't open controllers by default
 ----------------------  -------------------------------------------------
 --locale LOCALE         Use a custom locale for loading translations
-                        (e.g 'fr')
+                        (e.g ``fr``)
 ----------------------  -------------------------------------------------
 -f, --fullScreen        Starts Mixxx in full-screen mode
 ----------------------  -------------------------------------------------
+--logLevel LEVEL        Sets the verbosity of command line logging
+
+                        * ``critical`` - Critical/Fatal only
+                        * ``warning``  - Above + Warnings
+                        * ``info``     - Above + Informational messages
+                        * ``debug``    - Above + Debug/Developer messages
+                        * ``trace``    - Above + Profiling messages
```

Comment:
--logFlushLevel LEVEL   

Sets the the logging level at which the log buffer is flushed to mixxx.log. LEVEL is one of the values defined at --logLevel above.